gulpfile.js

```diff
@@ -3,12 +3,17 @@ var gulp = require('gulp'),
     uglify = require('gulp-uglify'),
     rename = require('gulp-rename'),
     ngAnnotate = require('gulp-ng-annotate'),
+    injectVersion = require('gulp-inject-version'),
     sourceFiles = [
       'angular-lock.js'
     ];
 
 gulp.task('build', function() {
   gulp.src(sourceFiles)
+    .pipe(injectVersion({
+      prepend: '',
+      replace: /%%PACKAGE_VERSION%%/g
```

Comment:
Where does the replacement version number come from?

Reply:
I'm using https://www.npmjs.com/package/gulp-inject-version. Takes the `version` property from the `package.json` file in the root and replaces it globally wherever the placeholder is present.